packages/waku/src/client.ts

```diff
@@ -49,13 +49,11 @@ export const fetchRSC = cache(
   ): Elements => {
     const options = {
       async callServer(actionId: string, args: unknown[]) {
-        const response = fetch(
-          basePath + encodeInput(encodeURIComponent(actionId)),
-          {
-            method: 'POST',
-            body: await encodeReply(args),
-          },
-        );
+        const input = basePath + encodeInput(encodeURIComponent(actionId));
```

Comment:
In my terminology, `input` is a string passed to a function in `entries.ts`.
So, let's rename it:
```suggestion
        const reqPath = basePath + encodeInput(encodeURIComponent(actionId));
```